docs/api-guide/filtering.md

```diff
@@ -218,6 +218,13 @@ For example:
 
 By default, the search parameter is named `'search`', but this may be overridden with the `SEARCH_PARAM` setting.
 
+To dynamically change search fields based on request content, it's possible to subclass the `SearchFilter` and override the `get_search_fields()` function. For example, the following subclass will only search on `title` if the query parameter `title_only` is in the request:
+
+    class CustomSearchFilter(self, view, request):
```

Comment:
Did you mean:

```
class CustomSearchFilter(SearchFilter):
    def get_search_fields(self, view, request):
      ...
```

Reply:
Thanks, someone else noticed this in #6487.

Reply:
Thanks for catching! Apologies for the mistake :(